spec/controllers/posts_controller_spec.rb

```diff
@@ -14,6 +14,34 @@
     expect(response).to have_http_status(200)
   end
 
+  describe 'posts aka conversations with pagination' do
```

Comment:
I think the only other test I would add is attempting to get a page beyond the number of posts. As in. add 3 posts, then try to get page 10 and ensure it doesn't blow up :)

Reply:
Thank you for the review @w1zeman1p! Good idea for one more test.  I'll add it 🗿👌🍭